core/plugins/org.fusesource.ide.help/contexts.xml

```diff
@@ -259,168 +250,254 @@
+    </context>
+    <context id="aws-kms">
+        <description>AWS KMS Component Options</description>
+        <topic href="aws-kms-component.html" label="AWS KMS"/>
+    </context>
+    <context id="aws-lambda">
+        <description>AWS Lambda Component Options</description>
+        <topic href="aws-lambda-component.html" label="AWS Lambda"/>
+    </context>
+    <context id="aws-mq">
+        <description>AWS MQ Component Options</description>
+        <topic href="aws-mq-component.html" label="AWS MQ"/>
     </context>
-    
     <context id="aws-s3">
-        <description>AWS-S3 Component Options</description>
-        <topic href="aws-s3-component.html" label="AWS-S3"/>
+        <description>AWS S3 Storage Service Component Options</description>
+        <topic href="aws-s3-component.html" label="AWS S3 Storage Service"/>
     </context>
     <context id="aws-sdb">
-        <description>AWS-SDB Component Options</description>
-        <topic href="aws-sdb-component.html" label="AWS-SDB"/>
+        <description>AWS SimpleDB Component Options</description>
+        <topic href="aws-sdb-component.html" label="AWS SimpleDB"/>
     </context>
     <context id="aws-ses">
-        <description>AWS-SES Component Options</description>
-        <topic href="aws-ses-component.html" label="AWS-SES"/>
+        <description>AWS Simple Email Service Component Options</description>
+        <topic href="aws-ses-component.html" label="AWS Simple Email Service"/>
     </context>
     <context id="aws-sns">
-        <description>AWS-SNS Component Options</description>
-        <topic href="aws-sns-component.html" label="AWS-SNS"/>
+        <description>AWS Simple Notification System Component Options</description>
+        <topic href="aws-sns-component.html" label="AWS Simple Notification System"/>
     </context>
     <context id="aws-sqs">
-        <description>AWS-SQS Component Options</description>
-        <topic href="aws-sqs-component.html" label="AWS-SQS"/>
+        <description>AWS Simple Queue Service Component Options</description>
+        <topic href="aws-sqs-component.html" label="AWS Simple Queue Service"/>
     </context>
     <context id="aws-swf">
-        <description>AWS-SWF Component Options</description>
-        <topic href="aws-swf-component.html" label="AWS-SWF"/>
+        <description>AWS Simple Workflow Component Options</description>
+        <topic href="aws-swf-component.html" label="AWS Simple Workflow"/>
+    </context>
+    <context id="azure-blob">
+        <description>Azure Storage Blob Service Component Options</description>
+        <topic href="azure-blob-component.html" label="Azure Storage Blob Service"/>
+    </context>
+    <context id="azure-queue">
+        <description>Azure Storage Queue Service Component Options</description>
+        <topic href="azure-queue-component.html" label="Azure Storage Queue Service"/>
     </context>
-    <context id="beanEIP">
+    <context id="bean">
```

Comment:
why there is no more a beanEIP? When selecting the "Bean" on the diagram, it is currently searching for a "beanEIP" context ID. So cannot find it anymore, and the documentation page displayed is then the generic one:
![image](https://user-images.githubusercontent.com/1105127/38487920-1527e0d0-3be2-11e8-8595-2ea7489febce.png)

Reply:
I can fix this. I can update the context.html to add a context ID = "beanEIP" and point to the bean.html file. Should I just commit it to this same branch for this pull request?

Reply:
Yes, on same branch is perfect